### DISCUSSION TITLE
HLS / DASH streams

### DISCUSSION THREAD

**Discussion by z34pec** [Edzelf/ESP32Radio-V2#118] [General]
Looks like BBS has converted from classic, shoutcast, to new HLS / DASH technology of streaming it internet radio channels.
Old stream channels do not work and at the moment only BBC Worldservice is available in old format. All other BBC channels are switched to HLS / DASH format.
I tried to search for boards that can convert this type of streaming format but no luck.
Is it the beginning of the end of well known internet radio and products based on shoutcast technology ?
Would we be able to have any solution for this stream method, without using our phones, pc's or tablets ?

**Comment by chainsawdave** [comment]
Any updates on HLS/DAS streams? I've not managed to get them to play
